CMakeExt/MPI.cmake

```diff
@@ -35,3 +37,18 @@ if (MPI_INCLUDE_PATH AND MPI_LIBRARY)
 else (MPI_INCLUDE_PATH AND MPI_LIBRARY)
   set(MPI_FOUND FALSE CACHE BOOL "Did not find the MPI library")
 endif (MPI_INCLUDE_PATH AND MPI_LIBRARY)
+
+
+# check for MPI-3
+
+set(CMAKE_REQUIRED_INCLUDES ${MPI_INCLUDE_PATH})
```

Comment:
I'm just curious, what is the scope of this `set`? Is it bound to this `.cmake` file? Or should we reset it after the `CHECK_SYMBOL_EXISTS` call. Neither the documention nor the `CheckSymbolExists.cmake` file gives me a clue about this.